error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/MapRulesTestInput.java

```diff
@@ -44,4 +44,15 @@ Stream<String> testMapKeyStream() {
   Stream<Integer> testMapValueStream() {
     return ImmutableMap.of("foo", 1).entrySet().stream().map(Map.Entry::getValue);
   }
+
+  Stream<boolean> testMapIsEmpty() {
```

Comment:
This wasn't flagged because of a tiny line of code missing in the base PR 😬. Will drop this method :).

Reply:
Thanks, overlooked that